packages/console/src/hooks/use-enabled-connector-types.ts

```diff
@@ -0,0 +1,26 @@
+import type { ConnectorResponse, ConnectorType } from '@logto/schemas';
+import { useCallback, useMemo } from 'react';
+import useSWR from 'swr';
+
+import type { RequestError } from './use-api';
+
+const useEnabledConnectorTypes = () => {
+  const { data: connectors } = useSWR<ConnectorResponse[], RequestError>('/api/connectors');
+
+  const enabledConnectorTypes = useMemo(
+    () => connectors?.filter(({ enabled }) => enabled).map(({ type }) => type) ?? [],
+    [connectors]
+  );
+
+  const isConnectorTypeEnabled = useCallback(
+    (connectorType: ConnectorType) => enabledConnectorTypes.includes(connectorType),
+    [enabledConnectorTypes]
+  );
```

Comment:
Is this function needed? I saw you didn't use this function in code below

Reply:
Cool, a mistake.